src/IFrameNavigator.ts

```diff
@@ -1023,12 +1023,6 @@ export default class IFrameNavigator implements Navigator {
         body.style.fontSize = fontSize;
         body.style.lineHeight = "1.5";
 
-        // Disable text selection as we can’t handle this correctly anyway
```

Comment:
Do we know what the original reason for this was / what "can't handle this correctly" meant? Not that I think this is an issue just wondering because perhaps there is a specific bug that we should keep an eye out for. (maybe specific to mobile devices?)

Reply:
The commit that added it was to "Make status messages non-selectable", so maybe it was overkill?

Reply:
hmm, yeah, if that's the case it would seem that way to me

Reply:
A good thing to keep an eye out for anyway!